src/utils/generate_structs.jl

```diff
@@ -19,7 +19,7 @@ This file is auto-generated. Do not edit.
 
 # Arguments
 {{#parameters}}
-- `{{name}}::{{{data_type}}}`{{#comment}}: {{{comment}}}{{/comment}}{{#valid_range}}, validation range: `{{{valid_range}}}`{{/valid_range}}{{#validation_action}}, action if invalid: `{{{validation_action}}}`{{/validation_action}}
+- `{{name}}::{{{data_type}}}`{{#comment}}: {{{comment}}}{{/comment}}{{#default}}, default value: `{{{default}}}`{{/default}}{{#valid_range}}, validation range: `{{{valid_range}}}`{{/valid_range}}{{#validation_action}}, action if invalid: `{{{validation_action}}}`{{/validation_action}}
```

Comment:
Can we do something slightly different with `, default value`? It looks weird when the comment ends with a period. My original suggestion was to put this just before the comment.

Reply:
@jd-lara @daniel-thom  How's this, in front in parentheses?

Also removed the ending periods https://github.com/NREL-Sienna/PowerSystems.jl/pull/1111/commits/a23061991537da8536d887fed5ae169bdc76e05b 

We could also remove the `action if invalid` to shorten the strings, since the user will be able to see the warning or error anyways

![Screenshot 2024-06-04 at 5 04 19 PM](https://github.com/NREL-Sienna/InfrastructureSystems.jl/assets/12451220/d7d2f5a2-f4bf-4a2c-80a3-eb24d50be824)

Reply:
Looks good to me. I agree with your point about `action if invalid`. I doubt anyone relies on that.

Reply:
Removed action if invalid below